dbt/include/trino/macros/materializations/incremental.sql

```diff
@@ -1,17 +1,73 @@
-{% macro dbt_trino_get_append_sql(tmp_relation, target_relation) %}
+{% macro dbt_trino_validate_get_incremental_strategy(raw_strategy) %}
+  {#-- Validate the incremental strategy #}
+
+  {% set invalid_strategy_msg -%}
+    Invalid incremental strategy provided: {{ raw_strategy }}
+    Expected one of: 'append', 'insert_overwrite'
+  {%- endset %}
+
+  {% if raw_strategy not in ['append', 'insert_overwrite'] %}
+    {% do exceptions.raise_compiler_error(invalid_strategy_msg) %}
+  {% endif %}
+
+  {% do return(raw_strategy) %}
+{% endmacro %}
+
+
+{% macro dbt_trino_get_insert_overwrite_sql(source_relation, target_relation, unique_key, dest_columns) %}
+
+    {% if unique_key %}
+    delete from {{ target_relation }}
+    where ({{ unique_key }}) in (
+        select ({{ unique_key }})
+        from {{ source_relation }}
+        );
+    {% endif %}
+
+    {{ dbt_trino_get_insert_into_sql(source_relation, target_relation, dest_columns) }}
+{% endmacro %}
+
+
+{% macro dbt_trino_get_insert_into_sql(source_relation, target_relation, dest_columns) %}
+
+    {%- set dest_cols_csv = get_quoted_csv(dest_columns | map(attribute='name')) -%}
 
-    {%- set dest_columns = adapter.get_columns_in_relation(target_relation) -%}
-    {%- set dest_cols_csv = dest_columns | map(attribute='quoted') | join(', ') -%}
     insert into {{ target_relation }}
-    select {{dest_cols_csv}} from {{ tmp_relation.include(database=true, schema=true) }};
```

Comment:
Why have you changed `tmp_relation.include(database=true, schema=true)` to `source_relation.include(database=false, schema=false)` ? 

I believe `(database=false, schema=false)` causes tests to fail.

Reply:
I haven't, these were changes from the branch I forked `feature/incremental` .

Reply:
Probably you're just missing changes from https://github.com/starburstdata/dbt-trino/pull/39. 
Try to change `(database=false, schema=false)` to `(database=true, schema=true)`

Reply:
Yeah that works great